.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,49 @@
+# Whether or not squash-merging is enabled on this repository.
+# Defaults to `true`
+rebaseMergeAllowed: false
+
+# Whether or not rebase-merging is enabled on this repository.
+# Defaults to `true`
+squashMergeAllowed: true
```

Comment:
The comment for these two keys appear to be flipped

Reply:
Yes, will fix in the example README for sync-repo-settings bot. https://github.com/googleapis/repo-automation-bots/pull/774

Reply:
:+1: